### PR TITLE
Fix: Confirmation component accessibility (DFCT0010064)

### PR DESCRIPTION
**Pull request by ericenns**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

Fixes accessibility issue in Confirmation component that was caused by a form input not having a corresponding label.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/views/projects/samples/index.html.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
         data: {
           turbo_stream: true,
           controller: "action-link",
-          action_link_required_value: 1
+          action_link_required_value: 1,
```
Comment (deepsidhu85) [inline app/views/projects/samples/index.html.erb:23]: The comma at the end of the line here and below (line 33 and 51) shouldn't be necessary

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
This looks great, just have one comment below. Just need to test it out

File: app/views/projects/samples/index.html.erb (hunk 1 above)
Comment (ericenns) [reply app/views/projects/samples/index.html.erb:23]: The updated `erb-formatter` is automatically adding these trailing commas now. See https://github.com/nebulab/erb-formatter/pull/35

Comment (deepsidhu85) [reply app/views/projects/samples/index.html.erb:23]: Thanks!

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**